packaging/nnstreamer.spec

```diff
@@ -2,6 +2,7 @@
 %define		gstpostfix	gstreamer-1.0
 %define		gstlibdir	%{_libdir}/%{gstpostfix}
 %define		nnstexampledir	/usr/lib/nnstreamer/bin
+%define		tensorflow-support	0
```

Comment:
Looks good. 
The **platform/upstream/tensorflow**  currently generates Tensorflow 1.09 package both armv7l and aarch64.  
Let's synchronize that the Tensorflow 1.09 package is not official.
* http://download.tizen.org/snapshots/tizen/unified/tizen-unified_20190718.4/repos/standard/packages/armv7l/tensorflow-v1.9.0-7.9.armv7l.rpm
* http://download.tizen.org/snapshots/tizen/unified/tizen-unified_20190718.4/repos/standard/packages/aarch64/tensorflow-v1.9.0-7.9.aarch64.rpm